### ISSUE TITLE
Fresh clone fails to run with a few errors

### ISSUE THREAD

**Issue by XapaJIaMnu** [browsermt/bergamot-translator-tests#9]
I did:
`make` and `./run_brt.sh`
I run into 2 issues:
```
/mnt/Storage/bergamot-translator/build_VSCODE/app/marian-decoder-new --help
terminate called after throwing an instance of 'CLI::CallForHelp'
  what():  This should be caught in your main function, see examples
  ```
  
Cause by:  https://github.com/browsermt/bergamot-translator-tests/blob/main/run_brt.sh#L56
And wrong path set for the model:
```
  [2021-03-10 13:42:36] Error: Model file does not exist: `/mnt/Storage/bergamot-translator/bergamot-translator-tests/models/deen/model.intgemm.alphas.bin
```

it should be:` /mnt/Storage/bergamot-translator/bergamot-translator-tests/models/deen/ende.student.tiny11/model.intgemm.alphas.bin`

The missing bit is: `model.intgemm.alphas.bin`

**Comment by jerinphilip** [comment]
* I get sane error outputs, which marian-dev (and build flags) are you using? Are exceptions disabled or anything by any chance? 
* The following is the Jenkins run, which seems to be building:

```
. /etc/environment
make install > install.log
make clean > clean.log
mkdir -p marian-dev && tar -C marian-dev -zxvf bergamot-translator.tgz
MARIAN=./marian-dev/build CUDA_VISIBLE_DEVICES=0,1 ./run_brt.sh && \
MARIAN=./marian-dev/build CUDA_VISIBLE_DEVICES=0,1 TIMEOUT=15m BRT_THREADS=40 BRT_EXPECTED_MAXTIME=500 ./run_brt.sh speed-tests/test_wngt20_perf.sh
```

Does it help setting MARIAN to an accurate path?

This could potential help uncover what caused a recent build failure (http://vali.inf.ed.ac.uk/jenkins/job/bergamot-translator-regression-tests/37/). I'm taking a look into this. Please feel free to respond to the rest.

**Comment by XapaJIaMnu** [comment]
it might be because i'm building the wasm stuff where exceptions are disabled (or something).

**Comment by jerinphilip** [comment]
Let's drag @abhi-agg into this who can (possibly) shed further light. 

```
/mnt/Storage/bergamot-translator/build_VSCODE/app/marian-decoder-new --help
terminate called after throwing an instance of 'CLI::CallForHelp'
  what():  This should be caught in your main function, see examples
```

Potential buggy-code (or is this desired?) path coming out of source currently being reviewed at https://github.com/browsermt/marian-dev/pull/24. Can't reproduce with vanilla.  Per our understanding, I don't do `-DUSE_WASM_COMPATIBLE_SOURCES=on`.

**Comment by jerinphilip** [comment]
Fresh clone errors should be solved in #11. Closing. @abhi-agg please address the exception if you know what's happening in marian-dev@wasm review.
